### PR TITLE
Automated rollout of Grafana Dashboards

### LINKED ISSUE #7026: Automated rollout of Grafana Dashboards
Integrate automated rollout of common Grafana dashboards to Grafana orgs managed by the MLA stack (e.g. Kubernetes, node-exporter, ...)

### PR DESCRIPTION
**Pull request by aborilov**
**What this PR does / why we need it**:
In this PR added automated rollout of common Grafana dashboards to Grafana orgs. OrgGrafanaController gets configmap with dashboards and adds those dashboards to all Grafana Orgs.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #7026 

**Special notes for your reviewer**:

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/controller/seed-controller-manager/mla/org_grafana_controller.go [hunk 1]
```diff
@@ -41,7 +43,8 @@ import (
 )
 
 const (
-	grafanaOrgAnnotationKey = "mla.k8c.io/organization"
+	grafanaOrgAnnotationKey        = "mla.k8c.io/organization"
+	grafanaDashboardsConfigmapName = "dashboards"
```
Comment (rastislavs) [inline pkg/controller/seed-controller-manager/mla/org_grafana_controller.go]: maybe rather `grafana-dashboards` ?

Comment (aborilov) [reply pkg/controller/seed-controller-manager/mla/org_grafana_controller.go]: done

File: pkg/controller/seed-controller-manager/mla/org_grafana_controller.go [hunk 2]
```diff
@@ -221,12 +255,29 @@ func (r *orgGrafanaController) createGrafanaOrg(ctx context.Context, expected gr
 	return expected, nil
 }
 
-func (r *orgGrafanaController) ensureOrganization(ctx context.Context, log *zap.SugaredLogger, project *kubermaticv1.Project, expected grafanasdk.Org, annotationKey string) error {
+func (r *orgGrafanaController) ensureDashboards(ctx context.Context, log *zap.SugaredLogger, orgID uint) error {
+	configMapList := &corev1.ConfigMapList{}
+	listOptions := &ctrlruntimeclient.ListOptions{Namespace: r.mlaNamespace}
+	if err := r.List(ctx, configMapList, listOptions); err != nil {
```
Comment (jiachengxu) [inline pkg/controller/seed-controller-manager/mla/org_grafana_controller.go]: actually, there is a helper function to list objects in namespace, and you can do something like:
```go
if err := r.List(ctx, configMapList, ctrlruntimeclient.InNamespace(r.mlaNamespace)); err != nil {
}
```

File: pkg/controller/seed-controller-manager/mla/org_grafana_controller_test.go [hunk 3]
```diff
@@ -62,7 +62,7 @@ func newTestOrgGrafanaReconciler(t *testing.T, objects []ctrlruntimeclient.Objec
 
 	grafanaClient := grafanasdk.NewClient(ts.URL, "admin:admin", ts.Client())
 
-	orgGrafanaController := newOrgGrafanaController(dynamicClient, kubermaticlog.Logger, grafanaClient)
+	orgGrafanaController := newOrgGrafanaController(dynamicClient, kubermaticlog.Logger, "mla", grafanaClient)
```
Comment (jiachengxu) [inline pkg/controller/seed-controller-manager/mla/org_grafana_controller_test.go:66]: Is it easy and possible to add some unit tests for dashboards?

File: pkg/controller/seed-controller-manager/mla/org_grafana_controller.go (hunk 2 above)
Comment (aborilov) [reply pkg/controller/seed-controller-manager/mla/org_grafana_controller.go]: oh, cool

File: pkg/controller/seed-controller-manager/mla/org_grafana_controller_test.go (hunk 3 above)
Comment (aborilov) [reply pkg/controller/seed-controller-manager/mla/org_grafana_controller_test.go:66]: yep, added test

File: pkg/controller/seed-controller-manager/mla/org_grafana_controller.go [hunk 4]
```diff
@@ -30,6 +31,10 @@ import (
 	"k8c.io/kubermatic/v2/pkg/kubernetes"
 	"k8c.io/kubermatic/v2/pkg/version/kubermatic"
 
+	predicateutil "k8c.io/kubermatic/v2/pkg/controller/util/predicate"
```
Comment (jiachengxu) [inline pkg/controller/seed-controller-manager/mla/org_grafana_controller.go]: can you please sort the imports?

Comment (aborilov) [reply pkg/controller/seed-controller-manager/mla/org_grafana_controller.go]: done

**Pull request merged**